src/genie/libs/parser/iosxe/tests/test_show_stackwise_virtual.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+from unittest.mock import Mock
+
+from genie.metaparser.util.exceptions import SchemaEmptyParserError
+from genie.libs.parser.iosxe.show_stackwise_virtual import ShowStackwiseVirtual
+
+
+# ======================================
+# Unit test for 'show stackwise-virtual'
+# ======================================
+class test_show_stackwise_virtual(unittest.TestCase):
+    """Unit test for 'show stackwise-virtual'"""
+
+    maxDiff = None
+    empty_output = {"execute.return_value": ""}
+    golden_parsed_output1 = {
```

Comment:
based on below documentation, different `stackwise virtual link` is possible on same switch.
https://www.cisco.com/c/en/us/td/docs/switches/lan/catalyst3850/software/release/16-6/command_reference/b_166_3850_cr/b_166_3850_new_chapter_01011.html#wp1144390350

So, I would suggest to update the schema like below.
```
{
    "domain": 100,
    "enabled": True,
    "switches": {
        1: {
            "stackwise_virtual_link": {
                1: {
                    "ports": ['HundredGigE1/0/3', 'HundredGigE1/0/4']
(snip)

Reply:
Are lists preferred in this repo? I see a lot of `index` key's with values of integers.

Reply:
it's good question. if they are in the middle of hierarchy, we have those with `index` if needed. but in this case, they are leafs (last elements of tree/hierarchy). so we usually have them as `list` if no other items go under those ports.

Reply:
Can you refer me to an example? I do not know how the schema should look like in this scenario

Reply:
The schema would be like below.
```
    schema = {
        "domain": int,
        "enabled": bool,
        Optional("switches"): {
            int: {
                "stackwise_virtual_link": {
                    int: {
                        "ports": list
                    }
                }
            }
        }
    }